src/main/java/bio/terra/workspace/common/utils/BaseStatusService.java

```diff
@@ -33,13 +37,18 @@ public BaseStatusService(long staleThresholdMillis) {
     currentStatus = new ApiSystemStatus().ok(false);
     lastUpdatedTimestampMillis = 0;
     this.staleThresholdMillis = staleThresholdMillis;
+    this.scheduler = Executors.newScheduledThreadPool(1);
+  }
+
+  @PostConstruct
+  public void startStatusChecking() {
+    scheduler.scheduleAtFixedRate(this::checkSubsystems, 1, 1, TimeUnit.MINUTES);
```

Comment:
this has a full 1-minute delay before starting checks, is that what we want? You could do something like
```
scheduler.scheduleAtFixedRate(this::checkSubsystems, 5, 60, TimeUnit.SECONDS)
```
to make the first status check happen sooner.

Reply:
Good point. That would actually delay readiness, the way things are setup.

Reply:
I considered plumbing these in as parameters, but didn't want to bother if we end up ditching this style of checking. I made a PR to track the work-not-done: PF-700